cassandra/src/main/scala/org/apache/zeppelin/cassandra/DisplaySystem.scala

```diff
@@ -507,11 +558,15 @@ object MetaDataConverter {
         .map(c => new ColumnDetails(c.columnMeta.getName, StaticColumn, c.columnMeta.getType))
     }
 
-    protected def convertClusteringColumns(columns: List[ColumnMetaWrapper], orders: List[DriverClusteringOrder]): List[ColumnDetails] = {
+    protected def convertClusteringColumns(
+      columns: List[ColumnMetaWrapper],
+      orders: List[DriverClusteringOrder]
+    ): List[ColumnDetails] = {
```

Comment:
I agree with you. it will be like this.
![screen shot 2016-05-30 at 16 14 57](https://cloud.githubusercontent.com/assets/648508/15643796/d5283312-2681-11e6-8902-8abfa7747872.png)

see alse the scala style guide, http://docs.scala-lang.org/style/declarations.html